src/components/header/header.js

```diff
@@ -12,9 +12,7 @@ class HeaderComponent extends LitElement {
 
       <header>
 
-        <a href="https://projectevergreen.github.io/">
-          <img src=${logo} alt="Project Evergreen logo"/>
-        </a>
+        <a href="https://projectevergreen.github.io/" title="Project Evergreen home link" style="background-image: url(./${logo})">&nbsp;</a>
```

Comment:
This needs to be `background-image` or it clobbers the `background-*` styles declared in the CSS file.